test/helper/atlas/waiters.go

```diff
@@ -27,7 +27,7 @@ func WaitForAtlasDeploymentStateToNotBeReached(ctx context.Context, atlasClient
 				}
 			}
 
-			g.Expect(allTrue).To(g.BeFalse())
+			Expect(allTrue).To(BeFalse())
```

Comment:
Was this method ever accepting the `g Gomega`?

Reply:
it was an alias in the module import

Reply:
Ok, I see. Just wanted to make sure it is not used in the `Eventually` blocks without accepting a `Gomega`